extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -42,6 +58,76 @@ function TrackedCornerstoneViewport({
     };
   }, []);
 
+  useEffect(() => {
+    if (!element) {
+      return;
+    }
+    const allTools = cornerstoneTools.store.state.tools;
+    const toolsForElement = allTools.filter(tool => tool.element === element);
+
+    toolsForElement.forEach(tool => {
+      if (
+        tool instanceof ArrowAnnotateTool ||
+        tool instanceof BidirectionalTool ||
+        tool instanceof EllipticalRoiTool ||
+        tool instanceof LengthTool
+      ) {
+        const configuration = tool.configuration;
+
+        configuration.renderDashed = !isTracked;
+
+        tool.configuration = configuration;
+      }
+    });
+
+    const enabledElement = cornerstone.getEnabledElement(element);
+
+    if (enabledElement.image) {
+      cornerstone.updateImage(element);
+    }
+  }, [isTracked]);
```

Comment:
This runs initially, and each time this prop is updated.
Because we set local state, that will trigger a re-render. I think it'd be possible to do this by only setting this initially, and then running the same logic each time we get a `subscription` event from the tracking service (without using internal state to trigger)

We can keep for now. Small refactor item that sits outside of the sprint?

Reply:
> Small refactor item that sits outside of the sprint?

Will make :+1:.